### ISSUE TITLE
*Staff of Levity* not working

### ISSUE THREAD

**Issue by Philosyang** [SleepyTrousers/EnderIO#4689]
#### Issue Description:

*Staff of Levity* is not working as intended.

#### What happens:

Make one *Staff of Levity* in survival mode, empowered it at least once, Shift + right click, nothing happens.

#### What you expected to happen:

*Applies anti-gravity to the player with Shift-R-Click*

#### Steps to reproduce:

1. Make one *Staff of Levity* in survival mode.
2. Empower it at least once (if not empowered then it makes sense not to work).
3. Shift + right click.
4. Nothing happens.

____
#### Affected Versions (Do *not* use "latest"):

- EnderIO: [5.0.25](https://minecraft.curseforge.com/projects/ender-io/files/2567180)
- EnderCore: [0.5.30](https://minecraft.curseforge.com/projects/endercore/files/2567175)
- Minecraft: 1.12.2
- Forge: 14.23.4.2705

#### Your most recent log file where the issue was present: 

N/A, the game won't crash or misbehave.

**Comment by EpicSquid** [comment]
It needs vapour of levity to work. I will attach that to the description next version.

**Comment by tyler489** [comment]
It also probably wont work if the player has some sort of creative like flight. Im assuming it uses levitation code?

**Comment by EpicSquid** [comment]
its literally the levitation potion effect

**Comment by tyler489** [comment]
Yep then it wont work with things that have have creative flight.
